articles/tech-resources/a-b-testing-sparkpost.md

```diff
@@ -9,31 +9,42 @@ An A/B test, in summary, is a common method of comparing templates to see which
+
+You can get your results using webhook events. For transmissions that used an A/B Test, Open and Click events will have the A/B Test ID, and the A/B Test Version number so you can look at the results.
+
+**Planned Enhancements and Known Issues**
+ *  The current "Learning Mode" does not automatically pick a winner. It provides real-time results via webhooks to allow users to analyze the results and make a determination which template is the best based on open or click rates. It is then a separate call to the A/B Testing API to change the default template to be the winning template. The next enhancement is the introduction of a statistical model in Bayesian Mode that will pick the winner automatically for you. This is coming soon!!
+ 
+ * An user interface (UI) is coming to allow non-technical users, such as Email Marketers and Email Product Managers to set up and manage A/B tests and get results. 
+ 
+ * Current functionality only supports single-recipient Transmissions REST API calls. Multiple recipient Transmissions calls that attempt to call an A/B Test will get an error. A/B testing is not supported for SMTP API
+
 
 **Notes about A/B testing specifics:**
 
-* A "group" in the context of A/B testing is the combination of a template and the recipients assigned to receive that template.
-* The winning template is automatically selected when the expiration time of the A/B test is reached. There are no manual controls to select a winning template. 
-* The winning template is chosen by best open rate across all groups. If there is a tie in open rate among templates being tested, a random winner will be selected from the templates that tied. A GET of the A/B test's ID through the API will list the template winner.
 * A minimum of 2 templates is required to run an A/B test. You may have up to a maximum of 10 templates in a single A/B test.
-* A recipient list used in an A/B test can have a **maximum** of 2,000 recipients.
-* The assignment of individual recipients into groups is randomized. You can pull which recipients were included in each group by performing a GET call with the A/B test's ID to the API endpoint.
-* The total number of recipients in the recipient list *must* be greater than the sum of all recipients assigned to test groups. There must be *at least 1* remaining recipient not belonging to any test group.
+* You can specify EITHER a population size with proportions or numbers of recipients to receive a given template variant or default OR you can specify and end time to end the test. 
```

Comment:
believe you have to specify:

1.  total population size (sample_size or percent)
2. end_time

so that is something like `You must specify EITHER a population size OR percentile breakdown for each template, AND an end_time to end the test`

Reply:
So they can specify both a population size AND and end time? And the first one reached is what ends the test?